prow/repoowners/repoowners.go

```diff
@@ -257,9 +258,23 @@ func (c *Client) LoadRepoOwners(org, repo, base string) (RepoOwner, error) {
 				entry.aliases = loadAliasesFrom(gitRepo.Dir, log)
 			}
 
-			dirBlacklist := commonDirBlacklist.Union(sets.NewString(c.ownersDirBlacklist().DirBlacklist(org, repo)...))
+			blacklistPatterns := append(c.ownersDirBlacklist().DirBlacklist(org, repo), commonDirBlacklist...)
```

Comment:
Why did we ditch the set?

Reply:
It's no longer useful. With string matching you could do `if o.dirBlacklist.Has(filename)`, but that's not going to work with regexps.